devices/src/virtio/block.rs

```diff
@@ -1513,8 +1522,8 @@ mod tests {
 
         // test block device update handler
         {
-            let f = NamedTempFile::new().unwrap();
-            let path = f.path().to_path_buf();
+            let f = TempFile::new("/tmp/").unwrap();
+            let path = f.as_path().to_path_buf();
```

Comment:
Can't we use here just `let path = f.as_path()`?

Reply:
Sure. Done that: [devices/src/virtio/block.rs:1527](https://github.com/firecracker-microvm/firecracker/blob/87231aa9c2462d00b58edf264d1a25482696281a/devices/src/virtio/block.rs#L1528)